working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<p>very correct, but why not choosing a more comprehensible name for the patients data? <code>dct</code>  is quite generic, isn't it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>